src/System.Collections/tests/Generic/LinkedList/LinkedList.Generic.Tests.Constructor.cs

```diff
@@ -236,7 +236,7 @@ private void VerifyGenericEnumerator(ICollection<T> collection, T[] expectedItem
             int iterations = 0;
             int expectedCount = expectedItems.Length;
 
-            //[] Verify non deterministic behavior of current every time it is called before a call to MoveNext() has been made			
+            //[] Verify non deterministic behavior of current every time it is called before a call to MoveNext() has been made            
```

Comment:
Shouldn't we remove the trailing tabs instead?

Reply:
I manually removed the ones I spotted.  But there's a ton of trailing whitespace in general, and I figured this wasn't making it any worse, and subsequently we could remove all of it programmatically.  In fact, stylecop rule SA1028 looks for exactly that, and so this can be addressed if/when that's enabled.

Reply:
Sounds good 😄